dissect/target/helpers/configutil.py

```diff
@@ -31,9 +32,20 @@
     import yaml
 
     PY_YAML = True
-except (AttributeError, ImportError):
+except ImportError:
     PY_YAML = False
 
+try:
+    if sys.version_info < (3, 11):
+        import tomli as toml
+    else:
+        # tomllib is included since python 3.11
+        import tomllib as toml
+
+    TOML = True
+except ImportError:
+    TOML = False
```

Comment:
```suggestion
    HAS_TOML = True
except ImportError:
    HAS_TOML = False
```
To be consistent with other places where we have this pattern.

Reply:
The PY_YAML one will be handled with https://github.com/fox-it/dissect.target/pull/563/files